source/main.cpp

```diff
@@ -271,14 +268,9 @@ void GameLoop(PlayerInfo &player, const Conversation &conversation, const string
 
 	const bool isHeadless = (testContext.CurrentTest() && !debugMode);
 
-	// IsDone becomes true when the game is quit.
-	while(!menuPanels.IsDone())
+	auto ProcessEvents = [&menuPanels, &gamePanels, &player, &cursorTime, &toggleTimeout, &debugMode, &isPaused,
+			&isFastForward]
```

Comment:
Wow. No-parameter lambdas look funky. Could we at least make it `[&stuff]()`?
It would appear that this lambda pattern won't work in future C++ versions. ~~Or don't care because at our rate it'll be 2033 before we increase our C++ version again.~~
![image](https://github.com/endless-sky/endless-sky/assets/17688683/4f3312c3-b6f9-45c1-82c6-a2a79d41aadb)

Reply:
On the contrary, C++23 expands on parameter-less lambdas:

![image](https://github.com/endless-sky/endless-sky/assets/85879619/8e2c4d58-be07-45bb-abec-85dca03e1c5c)